packages/components/pie-form-label/src/form-label.scss

```diff
@@ -1 +1,33 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.pie-form-label {
+    --form-label-font-size: #{p.font-size(--dt-font-size-14)};
+    --form-label-line-height: calc(var(--dt-font-size-14-line-height) * 1px);
+    --form-label-font-weight: var(--dt-font-weight-bold);
+    --form-label-color: var(--dt-color-content-default);
+
+    display: flex;
+    justify-content: space-between;
+    gap: var(--dt-spacing-d);
+    width: 100%;
```

Comment:
display flex should make this behave like a block element, so I believe width: 100% may be redundant